hr/specs/swagger.yaml

```diff
@@ -154,7 +154,9 @@ definitions:
         type: boolean
       employee_id:
         type: integer
-      name:
```

Comment:
Change with plusses and minuses.

+ Streamlines the conversion between the database model and the api model
- Reduces visibility and adds 1 extra field to the response that is unecessary (because before it gets to the api model, fname and lname can be merged together after they come out of the database).

Overall it's okay. I think it'll help speed up our time to release (aka. less work).